src/Datadog.Trace/Configuration/IntegrationIds.cs

```diff
@@ -13,6 +13,7 @@ internal enum IntegrationIds
         AspNetMvc,
         AspNetWebApi2,
         GraphQL,
+        Kafka,
```

Comment:
Similar to this comment, can you add this to the end of the enums just in case: https://github.com/DataDog/dd-trace-dotnet/pull/1457#discussion_r630895274

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/dff1497fe9b3af8a43b35d72a32c5a91fa8b1fdc